### DISCUSSION TITLE
Canal4 TV [ES]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#384] [Show and tell]
### Channel ID

Canal4TV.es

### Stream URL

https://stream.castr.com/63516516d5d3d0036518a3cd/live_73194be0508a11edb25de99cc96ba4dc/index.m3u8

### Notes

Ref (i think): https://canal4tvtelde.com/web/en-directo

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://stream.castr.com/63516516d5d3d0036518a3cd/live_73194be0508a11edb25de99cc96ba4dc/index.m3u8 (offline)
<img width="780" alt="image" src="https://user-images.githubusercontent.com/7253922/232256250-df186c6b-d765-45fe-b0bf-e61111b2399e.png">
